main.go

```diff
@@ -6,6 +6,10 @@ import (
 	"github.com/joyent/containerpilot/core"
 
 	log "github.com/Sirupsen/logrus"
+
+	// Import backends so that they initialize
```

Comment:
I tried this, but it seems like blank imports are suppose to be imported in main package. (The linter complained). Also would cause a dependency loop: `discovery` -> `etcd` -> `discovery`